tests/org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/formatting/TypeReferenceFormatterTest.xtend

```diff
@@ -21,7 +21,7 @@ class TypeReferenceFormatterTest {
 				{
 					var �toBeFormatted� x
 				}
-			'''
+			'''.toString.trim
```

Comment:
The changed expectation looks like it shadows a changed behavior. Blocks will now trim a trailing hidden / ws?